serverless.yml

```diff
@@ -23,7 +23,7 @@ functions:
     handler: src/functions/updateUploadStatus/framework/handler.handler
     events:
       - http:
-          path: test-result/{app-ref}/upload
+          path: test-result/{appRef}/upload
```

Comment:
Changing this will mean you could not locally invoke this lambda - see here ```src/functions/updateUploadStatus/framework/handler.ts```

The changed name of the pathParameters will now fail
``` const appRefPathParam = event.pathParameters['app-ref'];```

Reply:
without this change its not possible to serve locally.
![image](https://user-images.githubusercontent.com/48550267/232409258-f553406b-ae6d-4461-816e-d4cf4b6407df.png)

Reply:
The 'serving' may no work, but this will mean the code will not as mentioned above.